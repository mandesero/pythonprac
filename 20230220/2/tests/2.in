down
down
left
addmon 7 7 Ops...
left
left
down
left
down
addmon 7 7 New Monster
up
right
