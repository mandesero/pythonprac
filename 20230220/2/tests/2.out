Moved to (0, 9)
Moved to (0, 8)
Moved to (9, 8)
Added monster to (7, 7) saying Ops...
Moved to (8, 8)
Moved to (7, 8)
Moved to (7, 7)
 ________ 
< Ops... >
 -------- 
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
Moved to (6, 7)
Moved to (6, 6)
Added monster to (7, 7) saying New Monster
Replaced the old monster
Moved to (6, 7)
Moved to (7, 7)
 _____________ 
< New Monster >
 ------------- 
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
